<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memos</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="listMemos">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>0</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="listPeers">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>110</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRefreshPeers">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>160</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSendMemo">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>160</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Memo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRecordMemo">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>80</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPlayMemo">
    <property name="geometry">
     <rect>
      <x>224</x>
      <y>80</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDeleteMemo">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>80</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelMemos">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Memos</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPeers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Peers</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>319</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMemos">
    <property name="title">
     <string>Memos</string>
    </property>
    <addaction name="actionRefreshMemos"/>
    <addaction name="actionRefreshPeers"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuMemos"/>
  </widget>
  <action name="actionRefreshMemos">
   <property name="text">
    <string>Refresh Memos</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionRefreshPeers">
   <property name="text">
    <string>Refresh Peers</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
